<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QMainWindow" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>866</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>메뉴</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>831</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ice cream</string>
     </attribute>
     <widget class="QGraphicsView" name="Vanila">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Choco">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Strawberry">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="minimumSize">
      <size>
       <width>827</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="title">
      <string>topping</string>
     </attribute>
     <widget class="QGraphicsView" name="topping1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QGraphicsView" name="topping_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="topping_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_Button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_Button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
